.gitignore

```diff
@@ -29,3 +29,7 @@ web/skins
 deployment/deployment.retry
 
 tmp/
+
+/build/database/*
+!/build/database/01.Database Schema.sql
+!/build/database/02.Default Data.sql
```

Comment:
Please use underscores instead of spaces. This will avoid the need for escaping when referring to these files on the command line.